tests/syscall/fs/enc/CMakeLists.txt

```diff
@@ -22,5 +22,21 @@ else()
     set(OESGXFSENCLAVE "")
 endif()
 
+target_compile_options(fs_enc PRIVATE
+    -Wno-error
+    -Wno-unknown-pragmas
+    -Wno-unused-function
+    -Wno-return-type
+    -Wno-unused-variable
+    -Wno-missing-braces
+    -Wno-unused-parameter
+    -Wno-float-conversion
+    -Wno-sign-conversion
+    -Wno-sign-compare
+    -Wno-conversion
+    -Wno-parentheses
+    -Wno-unused-value
+    -Wno-overflow)
```

Comment:
Why do we need to add all these options?

Reply:
@jhand2 this pull request is still under working. These options are added to make it more convenient to comment out some calls of functions.